libs/perun/namespace-password-form/src/lib/password-form/password-form.component.ts

```diff
@@ -40,7 +40,7 @@ export class PasswordFormComponent implements OnInit, OnChanges {
           const languageAttribute: Attribute = richUser.userAttributes.find(
             (att) => att.friendlyName === 'preferredLanguage'
           );
-          this.language = (languageAttribute?.value as unknown as string) ?? 'en';
+          this.language = (languageAttribute?.value as string) ?? 'en';
```

Comment:
transfer to string was here before eslint
`this.language =
            languageAttribute && languageAttribute.value
              ? languageAttribute.value.toString()
              : 'en';`